src/main/java/org/folio/circulation/support/request/RequestHelper.java

```diff
@@ -17,4 +30,23 @@ public static boolean isRequestBeganFulfillment(Request request) {
   public static boolean isPageRequest(Request request) {
     return request != null && request.getRequestType() == RequestType.PAGE;
   }
+
+  public static CompletableFuture<Result<List<List<String>>>> mapItemIdsInBatchItemIds(List<String> itemIds) {
```

Comment:
This method appears to be used for splitting up a list of IDs into smaller batches. How is this related to requests? I think this might be better kept in its own class rather than with request utilities.

Reply:
Moved